### DISCUSSION TITLE
xmake require --uninstall 后还有属性没有清除

### DISCUSSION THREAD

**Discussion by xqyjlj** [xmake-io/xmake#3719] [Q&A]
如题，在我执行 `xmake require --uninstall` 卸载包后，xmake确实已经删除了库文件，但是工程项目依然认为库还是安装了的，这时构建就会出现错误，因为库文件已经删了嘛，会直接报找不到文件

![image](https://user-images.githubusercontent.com/51477609/236783440-34b81304-2d74-4807-80e5-341a7934d259.png)

现在我是执行``xmake require --uninstall`后，再手动把.xmake文件夹删了，然后再次config，特别麻烦，希望能够解决

**Comment by waruqi** [comment]
自己 xmake f -c 清下cache，或者走 vscode 里面 xmake.onConfigure 重新配置下
